tests/affiliatedcertification/tests/affiliated_certification_operator.go

```diff
@@ -93,52 +136,38 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
 
 			err = globalhelper.ValidateIfReportsAreValid(
 				affiliatedcertparameters.TestCaseOperatorAffiliatedCertName,
-				globalparameters.TestCaseSkipped)
+				globalparameters.TestCaseFailed)
 			Expect(err).ToNot(HaveOccurred(), "Error validating test reports")
 		})
 
 	// 46697
 	It("two operators to test, one belongs to certified-operators organization and its version is certified,"+
```

Comment:
Can you re-name this TC to something shorter? 
two operators to test, one is certified another is not certified [netative]

Reply:
There are two different types of operators considered uncertified for this test:
- operators that are not in the certified-operators group in the Red Hat catalog
- operators that are in the certified-operators group but whose versions are not certified for the current ocp version

So there is a certain level of detail needed in the test case titles, but I can still try to make the ones like this shorter.

Reply:
Ok I see. In this case I would use something like:
two operators to test, the first is certified, the second is certified but has invalid version [netative]